addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/hardware/FritzahaWebInterface.java

```diff
@@ -53,13 +52,9 @@
      */
     protected String sid;
     /**
-     * HTTP client for asynchronous calls
+     * shared instance of HTTP client for asynchronous calls
      */
     protected HttpClient asyncclient;
-    /**
-     * Maximum number of simultaneous asynchronous connections
-     */
-    protected int asyncmaxconns = 20;
```

Comment:
Do you happen to know if that huge number was required? The shared client will actually only do 2 concurrent connections - we should be sure that this isn't an issue for this binding.

Reply:
The binding itself uses only three different destinations for communicating with the FRITZ!Box. One for polling and discovery and two for sending a user command to the FRITZ! interface. I assume no issues for this binding if we reduce the number of connections.

Reply:
ok, I assume you briefly tested it, so let's say it works :-)

Reply:
This version runs in my live environment for four days now w/o any problems.